src/catch.rs

```diff
@@ -188,11 +211,17 @@ fn match_single(
 fn match_multiple(
     line: &str,
     source: PackageSource,
-    command_regex: &str,
-    arg_regex: &str,
+    command_regex: &'static str,
```

Comment:
Why does it need to be a `'static` string?

Reply:
It's static, because lookup hashmap is static. It's a requirement for static variables